controllers/articles.controller.js

```diff
@@ -1,4 +1,12 @@
-const { fetchArticleById } = require("../models/articles.model");
+const { fetchArticles, fetchArticleById } = require("../models/articles.model");
+
+exports.getArticles = (request, response, next) => {
+  fetchArticles()
+    .then((articles) => {
+      response.send({ articles });
```

Comment:
I think at this point you should also send the status at this point :)

Reply:
It defaults to 200, I'm not a big fan of overwriting defaults with default values - they were set as default because they're so common & it saves time / improves neatness... Happy to listen to alternate POV here though?!

Reply:
I definitely agree here! One thing to consider is, would this be explicit? Even though it defaults to 200, if you have a younger less experienced dev, they might not be able to see it. Being explicit in what is happening will make it easier to identify bugs. It's really just more readability, but considering this is your own repo, without anyone else working on it, it's fine to leave it if you want :)

Reply:
Good point. I guess there are times when you want to be more explicit with certain functions, but if we had to start listing all default values explicitly the concept would quickly get ridiculous (I imagine require() for example uses about 1 million default values under the hood for example xD). As this is my repo I'll stick with it but good point about readibility!